feconf.py

```diff
@@ -339,6 +339,7 @@ def get_empty_ratings():
     u'14': 'about_oppia.yaml',
     u'15': 'fuzzy_exploration.yaml',
     u'16': 'all_interactions',
+    u'17': 'test_responsive_supplementary'
```

Comment:
Can we now remove this exploration, and optionally merge it into the test_interactions exploration (number 10)? I think that's a better long-term place for the tests here, and it helps avoid a proliferation of demo explorations.